src/main/java/picard/fingerprint/ExtractFingerprint.java

```diff
@@ -67,6 +76,16 @@
             "If argument EXTRACT_CONTAMINATION=true the resulting samplename will be \"<SAMPLE>-contamination\" (if not provided).", optional = true)
     public String SAMPLE_ALIAS = null;
 
+    @Argument(doc = "SAM/BAM or VCF file of uncontaminated sample of the same individual whose information we are trying to remove.  If we are extracting the fingerprint of the contaminating sample," +
+            " this should be a sample from the same individual as the contaminated sample.  If we are extracting the fingerprint of the contaminated sample, this should be a sample from the same " +
+            "individual as the contaminating sample.  If the sample being removed is a tumor sample, this sample should optimally be a normal sample from the same individual.", optional = true)
+    public File BACKGROUND = null;
+
+    @Argument(doc = "Whether the sample being removed (not the sample whose fingerprint is being extracted) is a tumor sample.  This argument can only be set to true if a BACKGROUND sample is included." +
+            "  In this case, the BACKGROUND sample will be assumed to be a normal sample from the same individual.  In order to avoid potential confounding due to LoH, all fingerprinting sites which genotype " +
+            "as het in the BACKGROUND sample will be removed from the output fingerprint.  If this argument is set to true and no BACKGROUND sample is provided the tool will throw an exception.", optional = true)
+    public boolean REMOVED_IS_TUMOR = false;
```

Comment:
would "BACKGROUND_IS_TUMOR" make sense here? Having trouble understanding the relationship between this arg and "BACKGROUND"

Reply:
I agree the naming here is confusing, but I don't think BACKGROUND_IS_TUMOR is quite right either.  What this flag indicates is that the sample which is mixed with the sample we are trying to fingerprint is a tumor.  The problem with "BACKGROUND_IS_TUMOR" is that the sample supplied in the BACKGROUND argument needs to be the associated normal, so I think that would make it more confusing.
Maybe "MIXED_WITH_TUMOR" would be clearer?

Reply:
So if the input sample is a contaminated that includes a mix of tumor and normal, the normal would be passed as BACKGROUND and this flag would be set to "true"?